src/components/index.ts

```diff
@@ -8,4 +8,4 @@ import BnTextarea from './BnTextarea/BnTextarea.vue';
 import BnFileInput from './BnFileInput/BnFileInput.vue';
 import BnCheckbox from './BnCheckbox/BnCheckbox.vue';
 
-export { Btn, BnInput, BnListbox, BnModal, BnToggle, BnPagination, BnTextarea, BnFileInput, BnCheckbox };
+export { BnBtn, BnInput, BnListbox, BnModal, BnToggle, BnPagination, BnTextarea, BnFileInput, BnCheckbox };
```

Comment:
This is now exporting `BnBtn`  but `src/lib/index.ts` is still importing/exporting Btn

Reply:
Corrected!